services/bettersocial/post/blockAnonymousPost.js

```diff
@@ -40,7 +40,6 @@ const BetterSocialBlockAnonymousPost = async (token, selfUserId, postId, source,
       message: e?.message || 'Error in fetching getstream post by post id'
     };
   }
-
   const authorAnonymousUserId = post?.actor?.id || null;
   console.log(`${selfAnonymousUserId?.user_id} vs ${authorAnonymousUserId}`);
   if (selfAnonymousUserId?.user_id === authorAnonymousUserId)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive user information like IDs directly to the console can be a security risk. Consider removing this or replacing it with a more generic message.
```diff
-   console.log(`${selfAnonymousUserId?.user_id} vs ${authorAnonymousUserId}`);
+   console.log('Comparing self and author anonymous user IDs');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->